WebDriverAgentLib/Routing/FBElementUtils.m

```diff
@@ -33,20 +50,40 @@ + (NSString *)wdAttributeNameForAttributeName:(NSString *)name
   return matchingTypes.copy;
 }
 
-+ (NSArray<NSString *> *)wdPropertyNames {
-  NSMutableArray *result = [NSMutableArray array];
-  unsigned int propsCount = 0;
-  objc_property_t *properties = protocol_copyPropertyList(objc_getProtocol("FBElement"), &propsCount);
-  for (unsigned int i = 0; i < propsCount; ++i) {
-    objc_property_t property = properties[i];
-    const char *name = property_getName(property);
-    NSString *nsName = [NSString stringWithUTF8String:name];
-    if (nsName && [nsName hasPrefix:WD_PREFIX]) {
-      [result addObject:nsName];
++ (NSDictionary<NSString *, id> *)wdProperties
+{
+  static NSDictionary *propertiesWithGetters;
+  static dispatch_once_t propGettersToken;
+  dispatch_once(&propGettersToken, ^{
+    NSMutableDictionary *result = [NSMutableDictionary new];
```

Comment:
If the only reason for this is fb_isVisible etc. I would consider unifying getter names so that we can avoid this.